From 26e5a2c4277c0a1a116daf8dfd913c0318034f07 Mon Sep 17 00:00:00 2001
From: Andre McCurdy <armccurdy@gmail.com>
Date: Mon, 4 Apr 2016 19:09:47 -0700
Subject: [PATCH] vc5/driver/V3DDriver.mk: dont strip libv3ddriver.so

Stripping libraries should happen during packaging, or as part of a
post-processing step when creating the rootfs, not when the binaries
are built (even for release builds).

Fix for QA warnings such as:

  WARNING: broadcom-refsw-15.2-r0 do_package: QA Issue: File '/usr/lib/libv3ddriver.so' from broadcom-refsw was already stripped, this will prevent future debugging! [already-stripped]

Signed-off-by: Andre McCurdy <armccurdy@gmail.com>
---
 rockford/middleware/vc5/driver/V3DDriver.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/rockford/middleware/vc5/driver/V3DDriver.mk b/rockford/middleware/vc5/driver/V3DDriver.mk
index c5df98b..ff2ae05 100644
--- a/rockford/middleware/vc5/driver/V3DDriver.mk
+++ b/rockford/middleware/vc5/driver/V3DDriver.mk
@@ -111,8 +111,6 @@ CFLAGS += -Os -DNDEBUG
 
 ifeq ($(PROFILING),0)
 CFLAGS += -fvisibility=hidden
-# Strip
-LDFLAGS += -s
 else
 CFLAGS += -g
 LDFLAGS += -g --export-dynamic
-- 
1.9.1

